src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx

```diff
@@ -620,13 +587,14 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
 
         return (
             <>
-                {!_.isEmpty(formError) && (
+                {formError && (
```

Comment:
LEt's cast to boolean here:
```suggestion
                {!!formError && (
```

[context](https://dev.to/maafaishal/avoid-operator-for-conditional-rendering-in-react-2de)